src/main/cli/settings.c

```diff
@@ -757,6 +757,9 @@ const clivalue_t valueTable[] = {
     { "rx_min_usec",                VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { PWM_PULSE_MIN, PWM_PULSE_MAX }, PG_RX_CONFIG, offsetof(rxConfig_t, rx_min_usec) },
     { "rx_max_usec",                VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { PWM_PULSE_MIN, PWM_PULSE_MAX }, PG_RX_CONFIG, offsetof(rxConfig_t, rx_max_usec) },
     { "serialrx_halfduplex",        VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_RX_CONFIG, offsetof(rxConfig_t, halfDuplex) },
+#if defined(USE_RX_MSP_OVERRIDE)
+    { "msp_override_channels_mask",      VAR_UINT32 | MASTER_VALUE, .config.minmaxUnsigned = {0, (1 << (uint32_t) MAX_SUPPORTED_RC_CHANNEL_COUNT) - 1}, PG_RX_CONFIG, offsetof(rxConfig_t, msp_override_channels_mask)},
```

Comment:
Heh, this needs to be `.config.u32Max = (1 << (uint32_t) MAX_SUPPORTED_RC_CHANNEL_COUNT) - 1` - `.config.minmaxUnsigned` is limited to `uint16_t`. (See the failing builds.)

Reply:
Thanks